caas/kubernetes/provider/k8s.go

```diff
@@ -327,6 +371,16 @@ func (k *kubernetesClient) EnsureNamespace() error {
 	return errors.Trace(err)
 }
 
+// CreateNamespace creates this broker's namespace.
+func (k *kubernetesClient) CreateNamespace(name string) error {
+	if name == "" {
```

Comment:
Better just to pass in the namespace we want - more explicit, easier to understand

Reply:
removed